src/patternfly/components/Nav/nav.scss

```diff
@@ -504,6 +543,10 @@
   position: relative;
   margin-top: var(--pf-c-nav__item--MarginTop);
 
+  &:last-child > .pf-c-nav__link {
+    --pf-c-nav__link--before--BorderWidth: var(--pf-c-nav__item--last-child__link--before--BorderColor);
```

Comment:
doesn't look like either of these vars are defined anywhere?

Reply:
Whoops